src/openzaak/components/zaken/models/zaken.py

```diff
@@ -548,6 +548,27 @@ class RelevanteZaakRelatie(models.Model):
         ),
     )
 
+    overige_relatie = models.CharField(
+        max_length=100,
+        verbose_name=_("overige relatie"),
+        blank=True,
+        null=True,
```

Comment:
Generally it's better to not make `CharField`s (or other string based fields) nullable, because this means that there can be two different values for "no data": an empty string `""` and `null` (see https://docs.djangoproject.com/en/4.2/ref/models/fields/#null)
```suggestion
```